demos/device_defender_for_aws/defender_demo.c

```diff
@@ -374,7 +392,9 @@ static BaseType_t collectDeviceMetrics( void )
 {
     BaseType_t status = pdFAIL;
     MetricsCollectorStatus_t metricsCollectorStatus;
-    uint32_t numOpenTcpPorts, numOpenUdpPorts, numEstablishedConnections;
+    uint32_t numOpenTcpPorts, numOpenUdpPorts, numEstablishedConnections, i;
+    UBaseType_t tasksWritten = { 0 };
+    TaskStatus_t pTaskStatus = { 0 };
```

Comment:
This is not a pointer type, so the `p` prefix is misleading.
```suggestion
    TaskStatus_t taskStatus = { 0 };
```